src/lib/customers/comet-group-en.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "Comet",
+	"title": "bespinian and Comet create the IoT health monitor BLOX",
+	"customer": "<a href=\"https://comet.tech/\" target=\"_blank\">Comet Group</a> is a Swiss company based in Flamatt employing more than 1700 people around the globe. They are a leading tech company driving innovation in the x-ray and radio frequency business. The division Comet x-ray develops and produces x-ray modules that enable non-destructive testing of materials and industry components. The inspected parts come from the aerospace, automotive, battery, semiconductors and other markets.",
+	"background": "The x-ray modules, Comet has deployed with their customers worldwide, require regular maintenance. Knowing what exactly is going on within these devices and how they are being used is crucial for efficiently working with them. With targeted data visualization and preventive alerts, one can recognize patterns, drifts, gaps and detect incorrect usage and prevent outages. That is why Comet wanted to innovate in this space and create such a tool called BLOX for them and their customers.",
```

Comment:
I think usually x-ray has the "X"  capitalised

Reply:
Thanks, @qasimwarraich. Fixed.